src/storage-client/src/metrics.rs

```diff
@@ -95,3 +98,15 @@ impl StatsCollector<ProtoStorageCommand, ProtoStorageResponse> for RehydratingSt
         }
     }
 }
+
+/// Metrics used by the `backpressure` operator.
+#[derive(Debug, Clone)]
+pub struct BackpressureMetrics {
+    /// A counter with the number of emitted bytes.
+    pub emitted_bytes: Arc<DeleteOnDropCounter<'static, AtomicU64, Vec<String>>>,
+    /// The last count of bytes we are waiting to be retired in the operator. This cannot
+    /// be directly compared to `retired_bytes`, but CAN indicate that backpressure is happening.
```

Comment:
OOC the reason it cannot be directly compared is because this is point in time vs that's a cumulative counter?

Reply:
yes!